src/sentry/static/sentry/app/components/onboardingWizard/todos.jsx

```diff
@@ -102,7 +109,8 @@ const TodoList = React.createClass({
       //   'skippable': false,
       //   'prereq': [1, 2, 5],
       //   'featureLocation': 'project',
-      //   'location': 'settings/user-reports/'
+      //   'location': 'settings/user-reports/',
+      // display: true
```

Comment:
Why is this commented out? Can we use `display: false` here if we want to keep it around?

Reply:
good point. afaik the only difference is that it will show up in the admin view which isn't a terrible thing

Reply:
Yup agreed. Now that we've added in "display" we should either use it here or drop this task altogether.